README.md

```diff
@@ -55,7 +95,7 @@ Spinal Cord MRI (T2 weighted, T2 star, MT) preprocessing include number of key s
     sct_deepseg_sc -i ${file}.nii.gz -c t2 -qc qc
```

Comment:
@Mfauziyya why add the commands if they are in the script? or specify that they run in the script, no need to rerun them